src/containers/FileExplorer/ImageFilesTab.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { Spin, Pagination } from 'antd';
+import { FilesMetadata } from '@cognite/sdk';
+import styled from 'styled-components';
+import moment from 'moment';
+import { RootState } from '../../reducers/index';
+import { selectApp, AppState } from '../../modules/app';
+import { sdk } from '../../index';
+
+const Images = styled.div`
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-wrap: wrap;
+  overflow: auto;
+  margin: -12px;
+  margin-bottom: 12px;
+
+  && > .item {
+    margin: 12px;
+    padding: 6px;
+    background: #fff;
+    flex: 1 250px;
+    transition: 0.3s all;
+  }
+
+  && > .item:hover {
+    background: #dfdfdf;
+  }
+
+  && > .item img {
+    width: 100%;
+    height: auto;
+  }
+
+  && > .item .image {
+    width: 100%;
+    height: 200px;
+    margin-bottom: 12px;
+    overflow: hidden;
+    background-size: cover;
+    background-position: center;
+  }
+  && > .item p {
+    margin-bottom: 0;
+  }
+`;
+
+type OrigProps = {};
+
+type Props = {
+  app: AppState;
+  onClickDocument: (file: FilesMetadata, index: number) => void;
+  searchResults: FilesMetadata[];
+  current: number;
+  setPage: (page: number) => void;
+  fetching: boolean;
+} & OrigProps;
+
+type State = {
+  imageUrls: { [id: number]: string | false };
+};
+
+class ImageFilesTab extends React.Component<Props, State> {
+  readonly state: Readonly<State> = {
+    imageUrls: {},
+  };
+
+  componentDidUpdate(prevProps: Props) {
+    if (
+      prevProps.current !== this.props.current ||
+      (this.props.searchResults.length > 0 &&
```

Comment:
Do we always have a `searchResults` in props?

Reply:
yep :) its required